auto_install/install.sh

```diff
@@ -2225,34 +2405,34 @@ confWireGuard(){
 
 	echo "::: Server Keys have been generated."
 
-	if [ "$pivpnenableipv6" == "1" ]; then
-
-	echo "[Interface]
-PrivateKey = $($SUDO cat /etc/wireguard/keys/server_priv)
-Address = ${vpnGw}/${subnetClass},${vpnGwv6}/${subnetClassv6}
-MTU = ${pivpnMTU}
-ListenPort = ${pivpnPORT}" | $SUDO tee /etc/wireguard/wg0.conf &> /dev/null
+	{
+		echo '[Interface]'
+		echo "PrivateKey = $(${SUDO} cat /etc/wireguard/keys/server_priv)"
+		echo -n "Address = ${vpnGw}/${subnetClass}"
 
-	else
+		if [ "$pivpnenableipv6" == "1" ]; then
+			echo ",${vpnGwv6}/${subnetClassv6}"
+		else
+			echo
+		fi
 
-	echo "[Interface]
-PrivateKey = $($SUDO cat /etc/wireguard/keys/server_priv)
-Address = ${vpnGw}/${subnetClass}
-MTU = ${pivpnMTU}
-ListenPort = ${pivpnPORT}" | $SUDO tee /etc/wireguard/wg0.conf &> /dev/null
+		echo "MTU = ${pivpnMTU}"
+		echo "ListenPort = ${pivpnPORT}"
+	} | ${SUDO} tee /etc/wireguard/wg0.conf &> /dev/null
 
-	fi
 	echo "::: Server config generated."
 }
 
 confNetwork(){
 	# Enable forwarding of internet traffic
-	$SUDO sed -i '/net.ipv4.ip_forward=1/s/^#//g' /etc/sysctl.conf
+	echo 'net.ipv4.ip_forward=1' | $SUDO tee /etc/sysctl.d/99-pivpn.conf > /dev/null
+
 	if [ "$pivpnenableipv6" == "1" ]; then
-		$SUDO sed -i '/net.ipv6.conf.all.forwarding=1/s/^#//g' /etc/sysctl.conf
-		echo "net.ipv6.conf.${IPv6dev}.accept_ra=2" | $SUDO tee /etc/sysctl.d/99-pivpn.conf > /dev/null
+		echo "net.ipv6.conf.all.forwarding=1
+net.ipv6.conf.eth0.accept_ra=2" | $SUDO tee -a /etc/sysctl.d/99-pivpn.conf > /dev/null
```

Comment:
My bad, I was testing just the snippet without variables.

```suggestion
net.ipv6.conf.${IPv6dev}.accept_ra=2" | $SUDO tee -a /etc/sysctl.d/99-pivpn.conf > /dev/null
```